### PR TITLE
Add license page

### LINKED ISSUE #555: Add license page
## Kind (Required)

- Improvement

 ## Overview (Required)

- There are options for this display method, such as using a library, but please choose freely. 😄 

 ## Links	

-

### PR DESCRIPTION
**Pull request by rkonno**
## Issue
- close #555 

## Overview (Required)
- Implemented license page

## Links
- https://developers.google.com/android/guides/opensource

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by rkonno** [comment]
At present, there is a problem that all dependent libraries are not enumerated.
I'm investigating this issue now.

I'd really appreciate it if you could give me your opinion.

File: feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt [hunk 1]
```diff
@@ -101,15 +103,20 @@ class AboutFragment : Fragment(R.layout.fragment_about), Injectable {
                 aboutItemFactory.create(
                     getString(R.string.about_item_licence)
                 ) {
-                    // TODO go licence-page
+                    OssLicensesMenuActivity.setActivityTitle(
+                        this.getString(R.string.licenses_label)
+                    )
+                    Navigation.findNavController(
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt]: Can we use `findNavController()`? 👀

**Comment by takahirom** [comment]
> At present, there is a problem that all dependent libraries are not enumerated.
I'm investigating this issue now.
I'd really appreciate it if you could give me your opinion.

This app is open source. So it doesn't matter if you can't enumerate. I think it is good to merge and create an issue.

**Comment by koji-1009** [comment]
Hi, the issue looks like to be related to the following issues.

<https://github.com/google/play-services-plugins/issues/7>
<https://github.com/google/play-services-plugins/issues/25>

 I am facing the same problem (in another project!), and want to know how to solve it!

File: feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt (hunk 1 above)
Comment (rkonno) [reply feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt]: Thanks! `findNavController()` can be usable.

**Comment by takahirom** [comment]
Looks good! How about remove WIP? And can you solve conflict? 🙇

**Comment by rkonno** [comment]
@takahirom 
Thanks! Resolved conflict and removed WIP.

**Comment by rkonno** [comment]
@koji-1009 
Thanks for sharing information related to the plugin issue!
I also created a new issue to solve this problem. (#713)

**Comment by takahirom** [comment]
On CI 👀 

```
Execution failed for task ':android-base:generateLicenses'.
> Ambiguous method overloading for method com.google.android.gms.oss.licenses.plugin.LicensesTask_Decorated#addLicensesFromPom.
  Cannot resolve which method to invoke for [null, class java.lang.String, class java.lang.String] due to overlapping prototypes between:
  	[class java.io.File, class java.lang.String, class java.lang.String]
  	[class java.lang.String, class java.lang.String, class java.lang.String]

```

**Pull request merged**